.github/ISSUE_TEMPLATE/4.docs-feedback.yml

```
@@ -0,0 +1,52 @@
+name: Docs feedback
+description: Improve documentation about MUI Core.
+labels: ['status: needs triage', 'docs']
+title: '[docs] '
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide a searchable summary of the issue in the title above ⬆️.
+
+        Thanks for contributing by creating an issue! ❤️
+  - type: checkboxes
+    attributes:
+      label: Duplicates
+      description: Please [search the history](https://github.com/mui/material-ui/issues) to see if an issue already exists for the same problem.
+      options:
+        - label: I have searched the existing issues
+          required: true
+
+  - type: input
+    id: page-url
```

This casing convention caught my attention, I was more expecting this https://dev.to/imichaelowolabi/what-case-should-your-api-request-response-be-ggo, thoughts?
```suggestion
    id: page_url
```

---

Right, maybe because it becomes an `id` attribute for which the kebab case is the convention. No preferences then.